components/HeroNotificationBox.tsx

```diff
@@ -0,0 +1,21 @@
+import Arrow from "../public/arrow_forward.svg";
+import Image from "next/image";
+export default function HeroNotificationBox() {
+  return (
+    <div className="heroNotificationBox">
+      <p className="text-2xl font-bold">We’re hosting an event!</p>
```

Comment:
We don't want to hard code content here, let's make this a prop instead.

Reply:
We might want a mock data provider of some sort then?

If we don't want to allocate resources to that, we can just allow placeholders until we have tickets to implement content management. Whatever dev that is can pipe the data down.

Reply:
yep, we can have it default to `We're hosting an event!` for now, but let's still make it a prop.

Reply:
Added the props, but still left the text as optional values in new commit